mesheryctl/pkg/utils/scripts.go

```diff
@@ -162,3 +162,44 @@ func GenerateConfigAKS(resourceGroup, clusterName string) error {
 
 	return generateCFG.Run()
 }
+
+
+// GenerateConfigEKS generates kube config file in /tmp/meshery/kubeconfig.yaml for a GKE cluster
+func GenerateConfigEKS(region, cluster string) error {
+	script := fmt.Sprintf(`
+	set -e
+	set -o pipefail
+
+	TARGET_FOLDER="/tmp/meshery/"
```

Comment:
@hexxdump can we avoid use of a temp location and use Meshery's application folder instead? `~/.meshery`

Reply:
Done @leecalcote

Reply:
Looking fine to me after the directory update. Need to test it out!